integration/kubernetes/k8s-projected-volume.bats

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+
+setup() {
+	export KUBECONFIG=/etc/kubernetes/admin.conf
+	if sudo -E kubectl get runtimeclass | grep -q kata; then
+		pod_config_dir="${BATS_TEST_DIRNAME}/runtimeclass_workloads"
+	else
+		pod_config_dir="${BATS_TEST_DIRNAME}/untrusted_workloads"
+	fi
+}
+
+@test "Projected volume" {
+	pod_name="test-projected-volume"
+
+	TMP_FILE=$(mktemp username.XXXX)
+	SECOND_TMP_FILE=$(mktemp password.XXXX)
+
+	# Create files containing the username and password
+	echo "admin" > $TMP_FILE
+	echo "1f2d1e2e67df" > $SECOND_TMP_FILE
```

Comment:
Nit: this looks remarkably like a commit ID. Plus this values is being used twice. I'd consider something like the following for clarity and to avoid duplication:

```sh
password="1f2d1e2e67df"
echo "$password" > $SECOND_TMP_FILE

   :
 
sudo -E kubectl exec $pod_name -- sh -c "$sec_check_cmd" | grep "$password"
```